lit_tests/kernel/wave/codegen.py

```diff
@@ -1941,6 +1949,10 @@ def test(a: tkl.Memory[M, N, ADDRESS_SPACE, tkl.f16]):
         # Testing reciprocal
         # %[[ONES:.+]] = arith.constant dense<1.000000e+00> : vector<4xf16>
         # %[[RECIPROCAL:.+]] = arith.divf %[[ONES]], %[[EXP2]] : vector<4xf16>
+        
+        # Testing abs
+        # %[[ABS:.+]] = math.absf %[[RECIPROCAL]]
+        # %[[ABS:.+]] = math.absi
```

Comment:
Can we renamed the floating abs to `%[[ABSF:.+]]` and int one to `%[[ABSI:.+]]`

Reply:
Fixed.

Reply:
thanks!